hawc/apps/mgmt/managers.py

```diff
@@ -50,71 +59,224 @@ def create_study_tasks(self, study):
         assessment = study.assessment
         if not assessment.enable_project_management:
             return
-        tasks = self._get_missing_tasks(study, assessment)
+
+        statuses = self._get_missing_statuses(assessment)
+        types = self._get_missing_types(assessment)
+        tasks = self._get_missing_tasks(study, assessment, statuses, types)
+
         logger.info(f"Creating {len(tasks)} tasks for study {study.id}.")
         self.bulk_create(tasks)
 
-    def _get_missing_tasks(self, study, assessment):
+    def _create_assessment_triggers(self, assessment, task_statuses, task_types):
+        """
+        Create task triggers for an assessment and save to database. This
+        should eventually take user-defined input, but currently creates all
+        possible defined triggers for managed assessments
```

Comment:
This function is a bit convoluted, but I anticipate changing it soon / in the next PR. Users are not able to add triggers on assessment creation yet, so the existing constants are used in ensuring triggers exist for new assessments. Thoughts on handling this?